### ISSUE TITLE
Edit Bio And Fork The Repo To Spread The Word

### ISSUE THREAD

**Issue by ferittuncer** [upend/IF_MS_BUYS_GITHUB_IMMA_OUT#104]
As GitHub censored our repo from trending list, I think we should think other ways of spreading the word.

One way would be editing our GitHub profile bios by appending "ms acquisition refugee" or whatever phrase you like that will explain the same thing. 

I just edited mine.

Also forking and pinning the repo helps to spread the word I think.

**Comment by fn-main** [comment]
We could make a r/programming post about it

**Comment by vassudanagunta** [comment]
thought let's not use "refugee". Even though it feels like the right metaphor, there's no reason to be disregard actual refugees who feel it trivializes what they go though. Please read the comments in #26

**Comment by moorchegue** [comment]
I edited the bio and forked this very repo to raise awareness. Taking the whole migration very gradually, but making it loud enough.

**Comment by ferittuncer** [comment]
I sent a pull request #112

**Comment by vassudanagunta** [comment]
I love this idea. Let's talk more about it. But first, please read Issue 114.

**Comment by vassudanagunta** [comment]
Hello,

The GitHub Evacuation Project has moved to GitLab (not an endorsement or even a final home). Your enthusiasm and contribution is still needed. Please check out [the new project home](https://gitlab.com/upend/github), and read [the project wiki](https://gitlab.com/upend/github/evac-guide/wikis) for info to get restarted.

Thank you!

vas

**Comment by vassudanagunta** [comment]
@ferittunce @infinitetismal @moorchegue This is of course a great idea. 

I believe a consistency of imagery and message has the best chance of catching fire, like the original repo did before we got buried from the trending list by Microsoft GitHub.

So if you guys would change your profile photo and add "github evacuee" to your profile and repos as I wrote under [Help move the movement](https://github.com/upend/IF_MS_BUYS_GITHUB_IMMA_OUT#help-move-a-movement), we'll have a seed number of people who've done that, and then I'll start promoting it on Twitter, and it can catch fire. 

[We already have a few](https://github.com/search?q=github+evacuee&type=Users), and that's without me even sending out any message.
